grid/language/named-entity-recognition/profile.supr

```diff
@@ -0,0 +1,125 @@
+"""
+Named-Entity Recognition
+Identify and classify named entities such as people, organizations, locations, dates, and other categories of words in a given text.
+"""
+name = "language/named-entity-recognition"
+version = "1.0.0"
+
+"""
+Named-Entity Recognition
+Identify and classify named entities such as people, organizations, locations, dates, and other categories of words in a given text.
+"""
+usecase NamedEntityRecognition safe {
+  input {
+    """
+    Text
+    The text you wish to identify and classify named entities from. Note that
+    most providers require the text to be less than 1000 characters
+    """
+    text! string!
+
+    """
+    Text Language
+    Optionally specify the exact language of the text. Useful for phrases
+    that might be ambigous.
```

Comment:
what is the format, maybe ad an example? ISO 639-1?

Reply:
Good point 👍 I added the format and example.

Here I thought about using enum as well but wasn't sure we want the profiles to carry 183-value enums?